### DISCUSSION TITLE
Is macos-latest updated to macos 12?

### DISCUSSION THREAD

**Discussion by raulcd** [actions/virtual-environments#6538] [General questions] [answered]
Hi,
I am using `macos-latest` for some of my runners and they seem to be using `macos-12` instead of `macos-11` as the `README` suggests:
https://github.com/actions/runner-images/blob/main/README.md#available-images

Is the `README` outdated? I can create a PR to update it if that's case.
Thanks
Raúl

**Comment by al-cheb** [comment, marked as answer]
Hey @raulcd.
Please take a look at - https://github.com/actions/runner-images/issues/6384

**Reply by assignUser** [reply]
A note about the migration in the README would be nice though :)

**Reply by al-cheb** [reply]
> A note about the migration in the README would be nice though :)

We have annotations:
![image](https://user-images.githubusercontent.com/47745270/200855739-3314ac5c-a3a7-4843-8a07-8c5746f3c630.png)

**Reply by assignUser** [reply]
Ah thanks! I missed  that as it only shows up on the summary and the issue was with a matrix build with only one sub-build failing :)
